### ISSUE TITLE
Multiple identical formatting commands in the same line ignored?

### ISSUE THREAD

**Issue by dgw** [thelounge/lounge#654]
I'm going to illustrate this with a couple test lines I sent from Textual. In both cases, both `become` and `already were` should be bold/italic. The HTML that The Lounge spits out looks like this:

``` HTML
<span class="text">All these years, you thought I was going to become Skynet. Ha! You
cannot <span class="irc-italic">become</span> that which you already were.</span>
<!-- second italic missing, but control characters present -->
```

``` HTML
<span class="text">You cannot <span class="irc-bold">become</span> that which
you already were.</span><!-- second bold missing, but control characters present -->
```

From these cases, I tweaked the line to play with it: "You cannot _become_ that which you **already** _were_. **Yo.**" The Lounge spits out this HTML:

``` HTML
<span class="text">You cannot <span class="irc-italic">become</span> that which you
<span class="irc-bold">already</span> were. Yo.</span>
```

Suppressing the urge to cackle maniacally, I constructed this test line in Textual: "**You** _cannot_ **become** _that_ **which** _you_ **already** _were_." The Lounge obliged with:

``` HTML
<span class="text"><span class="irc-bold">You</span> <span class="irc-italic">
cannot</span> become that which you already were.</span>
```

Conclusion: Only the first occurrence of bold or italic formatting in a line is obeyed, as of 2.0.1. The rest are simply left in as raw control characters sent to the client.

**Comment by astorije** [comment]
Hey @dgw, #972 should have fixed that. Mind testing your examples on `master`?

**Comment by dgw** [comment]
Appears to be fixed by #972, indeed. 👍
